onnxruntime/core/mlas/lib/mlasi.h

```diff
@@ -448,6 +448,18 @@ float
 
 typedef MLAS_REDUCE_MAXIMUM_FLOAT_KERNEL* PMLAS_REDUCE_MAXIMUM_FLOAT_KERNEL;
 
+typedef
+void
```

Comment:
Can this be called MLAS_REDUCE_MINIMUM_MAXIMUM_FLOAT_KERNEL? Or at least MLAS_REDUCE_MIN_MAX_FLOAT_KERNEL?